lib/index.js

```diff
@@ -37,6 +39,10 @@ exports.balm = (opts = {}) => {
 
     traversy(root, fileExtRegexStr, bundleFile);
 
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
I'd make an isDev variable of this check so it can be used down lower in the file too

Reply:
Done